Jenkinsfile_CNP

```diff
@@ -80,19 +80,21 @@ def uploadDmnDiagrams(String env, String changeID) {
   }
 }
 
+/*
 def checkForEnableHelmLabel(branch_name) {
   return new GithubAPI(this).getLabelsbyPattern(branch_name, "enableHelm").contains("enableHelm")
 }
+*/
 
 withPipeline(type, product, component) {
   pipelineConf = config
 
   onPR {
     env.ENVIRONMENT = "preview"
     loadVaultSecrets(secrets)
-    if (!checkForEnableHelmLabel(env.BRANCH_NAME)) {
+    /*if (!checkForEnableHelmLabel(env.BRANCH_NAME)) {
```

Comment:
shall we just remove it ? including the above commented method ? platops have removed the method fully so we wont be able to use it again. .if we required we can get from git history

Reply:
done